thrust/testing/async_copy.cu

```diff
@@ -10,6 +10,8 @@
 #  include <unittest/unittest.h>
 #  include <unittest/util_async.h>
 
+_CCCL_SUPPRESS_DEPRECATED_PUSH
+
```

Comment:
I am missing the `_CCCL_SUPRESS_DEPRECATEED_POP`

Reply:
IIRC, I omitted it on purpose to suppress warnings in the cudafe1.c file. But I should have added a comment about it.

Can we still go ahead, given this is the backport PR?

Reply:
I am good to go ahead. It's only a test source file.